File: OCH_Tao/ABC/A/131.js
const S = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0];
if (S[0] === S[1] || S[1] === S[2] || S[2] === S[3]) {
  console.log("Bad");
} else {
  console.log("Good");
}

File: OCH_Tao/ABC/A/131.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  string S;
  cin >> S;
  cout << (S[0] == S[1] || S[1] == S[2] || S[2] == S[3] ? "Bad" : "Good") << endl;
}